webpack.common.config.ts

```diff
@@ -192,17 +192,6 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
-    new HtmlWebpackPlugin({
```

Comment:
We need to remove some more part of this file, please check the issue description/steps!